pglookout/cluster_monitor.py

```diff
@@ -148,14 +154,15 @@ def connect_to_cluster_nodes_and_cleanup_old_nodes(self):
         for instance, connect_string in self.config.get("remote_conns", {}).items():
             self._connect_to_db(instance, dsn=connect_string)
 
-    def _standby_status_query(self, instance, db_conn):
-        """Status query that is executed on the standby node"""
+    def _query_cluster_member_state(self, instance, db_conn):
+        """Query a single cluster member for it's state"""
         f_result = None
         result = {"fetch_time": get_iso_timestamp(), "connection": False}
         if not db_conn:
             db_conn = self._connect_to_db(instance, self.config["remote_conns"].get(instance))
             if not db_conn:
                 return result
+        phase = None
```

Comment:
nit. I would instead move the next `phase = "querying status from"` out of the try, since the assignment won't fail (and if it did, it wouldn't fail with an exception type that is part of the except clause).